srv/modules/runners/validate.py

```diff
@@ -771,23 +771,18 @@ def _check_installed(self):
         install only happens once.
         """
         search = __utils__['deepsea_minions.show']()
-        results = self._silent_search(search, 'pkg.info_installed')
+        results = self._silent_search(search, 'pkg.list_pkgs')
 
-        for minion in results:
-            if isinstance(results[minion], dict) and self.package in results[minion]:
-                if 'version' in results[minion][self.package]:
-                    version = self._check_version(minion, 'pkg.info_installed',
-                                                  results[minion][self.package]['version'])
```

Comment:
Is this the only call to `_check_version`? If so, then maybe the entire `_check_version` function/method can be dropped as well?

Reply:
maybe more checks can be dropped here, this is \for now\ just to test the bad behavior of salt2019